agent/util-scripts/pbench-tool-meister-start

```diff
@@ -0,0 +1,450 @@
+#!/usr/bin/env python3
+# -*- mode: python -*-
+
+"""pbench-tool-meister-start
+
+Responsible for:
+
+   1. Starting a redis server
+   2. Loading up the tool group data for the target group into the redis server
+   3. Starting the tool-data-sink process
+   4. Starting all the local and remote tool meisters
+
+When complete we leave running locally, a redis server and a tool data sink
+process, and any local or remote tool meisters.
+
+The pbench-tool-meister-stop will take care of (gracefully) stopping all of
+these process, locally or remotely.
+"""
+
+import sys
+import json
+import os
+import time
+import errno
+import redis
+import logging
+
+# Port number is "One Tool" in hex 0x17001
+# FIXME: move to common area
+redis_port = 17001
+
+# Redis server configuration template for pbench's use
+redis_conf_tmpl = """bind {hostnames}
+daemonize yes
+dir {tm_dir}
+dbfilename pbench-redis.rdb
+logfile {tm_dir}/redis.log
+loglevel notice
+pidfile {tm_dir}/redis_{redis_port:d}.pid
+port {redis_port:d}
+"""
+
+# FIXME: this should be moved to a shared area
+channel = "tool-meister-chan"
```

Comment:
this should be configurable as well.

Reply:
Not sure want to have this configurable because that leaves the users in control of the successful operation.

We might want to add a per-instance prefix to it, but we can make such a change later.